src/main/scala/org/ergoplatform/modifiers/history/Header.scala

```diff
@@ -50,6 +51,8 @@ case class Header(version: Version,
 
   lazy val extensionId: ModifierId = BlockSection.computeId(Extension.modifierTypeId, id, extensionRoot)
 
+  override val minerPk: ECPoint = powSolution.pk
```

Comment:
Should be of CryptoConstants.EcPointType type (the same for the whole codebase)

Reply:
fixed